test/server/lib/request-verifier.js

```diff
@@ -0,0 +1,83 @@
+const test = require('tape')
+const crypto = require('../../../lib/crypto')
+const request = require('request')
+const requestVerifier = require('../../../server/lib/request-verifier.js')
+const serializer = require('../../../lib/serializer.js')
+serializer.initSerializer()
+const Express = require('express')
+
+test('users router', (t) => {
+  const app = Express()
+  app.param('userId', requestVerifier)
+  app.post('/:userId/', (_request, response) => { response.send('sweet') })
+
+  const server = app.listen(0, 'localhost', () => {
+    const serverUrl = `http://localhost:${server.address().port}`
+    console.log(`server up on ${serverUrl}`)
+
+    const keys = crypto.deriveKeys(crypto.getSeed())
+    const userId = Buffer.from(keys.publicKey).toString('base64')
+    const baseRequest = request.defaults({
+      baseUrl: `${serverUrl}/${encodeURIComponent(userId)}`
+    })
+
+    function signedTimestamp (secretKey, timestamp) {
+      if (!timestamp) { timestamp = Math.floor(Date.now() / 1000) }
+      const message = timestamp.toString()
+      return crypto.sign(serializer.serializer.stringToByteArray(message), secretKey)
```

Comment:
this needs to ensure that serializer is initialized; i handled this by putting the tests in the serializer.init callback in test/serializer.js